docs/components.md

```diff
@@ -103,14 +103,63 @@ To learn more about lifecycle methods, [see the lifecycle methods page](lifecycl
 
 ### Syntactic variants
 
+#### Closure components
+
+One of the easiest ways to manage state in a component is with a closure. A "closure component" is one that returns an object with a view function and optionally other lifecycle hooks. It has the ability to manage instance state within the body of the outer function.
+
+```javascript
+function ClosureComponent(initialVnode) {
+	// Each instance of this component has its own instance of `kind`
+	var kind = "closure component"
+
```

Comment:
we should standardize coding style — in "StableComponent" we don't leave a space between ivar and return

Reply:
The original [closureComponent example](https://github.com/MithrilJS/mithril.js/blob/next/docs/components.md#closure-components) had a space before the return. Personally that's how I write them, I find it easier to read. My preference would be to leave the space. So I'm going to add one to StableComponent. :)

BTW style-wise, I've also capitalized closure components as both class and POJOs were capitalized. It's also a common convention for factory functions (and extended to Typescript - frequently merged with the interface type returned by the factory.)